### PR TITLE
fix(react-devtools-inline): wrong type fix

### PR DESCRIPTION
**Pull request by Jack-Works**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/facebook/react/tree/main/packages/react-devtools-inline>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/react-devtools-inline/commons.d.ts [hunk 1]
```diff
@@ -43,13 +43,13 @@ export type FetchFileWithCaching = (url: string) => Promise<string>;
 export type Context = FetchFileWithCaching | null;
 
 export interface DevtoolsProps {
-    bridge: FrontendBridge;
+    bridge?: FrontendBridge;
```
Comment (YardenPorat) [inline types/react-devtools-inline/commons.d.ts:46]: not optional
image

File: types/react-devtools-inline/commons.d.ts [hunk 2]
```diff
@@ -43,13 +43,13 @@ export type FetchFileWithCaching = (url: string) => Promise<string>;
...
     browserTheme?: BrowserTheme | undefined;
     canViewElementSourceFunction?: CanViewElementSource | null | undefined;
     defaultTab?: TabID | undefined;
     enabledInspectedElementContextMenu?: boolean | undefined;
     showTabBar?: boolean | undefined;
-    store: Store;
+    store?: Store;
```
Comment (YardenPorat) [inline types/react-devtools-inline/commons.d.ts:52]: not optional
image

**Review by YardenPorat — CHANGES_REQUESTED** [review]

**Comment by Jack-Works** [comment]
@YardenPorat hi, `store` and `bridge` are actually optional because it is provided in the implementation, and the implementation of `initialize` includes the logic of providing default value.

https://github.com/facebook/react/blob/65e32e58b6057db1fdfed95a942fad4fc96da191/packages/react-devtools-inline/src/frontend.js#L101

**Review by YardenPorat — APPROVED** [review]

**Comment by Jack-Works** [comment]
Ready to merge

**Pull request merged**